### ISSUE TITLE
[주제2] Validation data 질문

### ISSUE THREAD

**Issue by rhksgud92** [DatathonInfo/H.D.A.I.2021#70]
안녕하세요... 학습결과 auc가 지나치게 높게 나와서 질문 드립니다.
혹시 한 환자에서 얻어지는 데이터가 validation그리고 test에 둘 다 섞여 들어간건 아닌가요?
overfitting 문제 인가해서 generalization을 할려고 validation 데이터를 여러개로 나눠서 test해봤는데도 auc가 지나치게 높습니다.

**Comment by wonyongchung** [comment]
@rhksgud92  저희 validation set AUC가 0.97정도 나오는데 혹시 얼마 정도 나오시나요?

**Comment by rhksgud92** [comment]
@wonyongchung 대부분 그 정도 나올 것 같아요! 부정맥이란게 원래 이렇게 쉬운 걸 수도 있고, 주최측에서 데이터를 잘못준비한것같아요. r-peak나 심전도 qrstp 정보값 같은거 고려 할 필요도 없네요

**Comment by DatathonInfo** [comment]
안녕하세요 참가자님,

질문하신 것처럼 Validation Set과 Train Set에 중복데이터는 없는 것으로 확인됩니다. 

기존에 나와있는 arrhythmia detection의 경우, 모델별로 차이는 있습니다만 
최근에 나온 논문을 보면 multiclass의 경우에도 0.91 ~ 0.98 정도의 AUC가 나오기도 합니다. 

필요하시면 [해당 논문](https://arxiv.org/pdf/2108.10226.pdf)도 함께 참고하시기 바라며, 
대회 잘 마무리하셔서 좋은 성과 있으시길 기원드립니다.

감사합니다.
H.D.A.I.2021 운영팀 드림
